infra/README.md

```diff
@@ -11,16 +11,22 @@
 
 ## Deploying the CDK
 To deploy the CDK stacks, you must have the appropriate permissions to create
-the CloudFormation stack and associated resources in a given AWS account. You 
-will also need:
-- A personal access token for the [Prow bot account](https://github.com/kubernetes/test-infra/blob/master/prow/getting_started_deploy.md#github-bot-account)
-- An HMAC token used to validate Github webhooks
-  - This can be generated using `openssl rand -hex 20`
+the CloudFormation stack and associated resources in a given AWS account.
+
+You will also need to **manually** create and configure a GitHub app for Prow as [documented here](https://github.com/kubernetes-sigs/prow/blob/main/site/content/en/docs/Legacy%20Snapshot/prow/getting_started_deploy.md#github-app).
+
+Once the GitHub app is configured, you will need three data elements from the app's settings page to pass to the CDK deployment:
+
+- The app ID
+- The app's private RSA key (in PEM format)
+- The app's configured webhook secret
+  - [**_NOTE_**: you can generate a valid value for this using `openssl rand -hex 20`]
 
 Use the following command to deploy the stack with the included requirements:
 ```bash
-export BOT_PAT='<bot personal access token>'
-export WEBHOOK_HMAC='<webhook HMAC>'
+export GITHUB_APP_ID='<GitHub app ID>'
+export GITHUB_APP_PRIVATE_KEY='<GitHub app private RSA key in PEM format>'
```

Comment:
nit: this can be a `cat` of the PEM file?

Reply:
Yes it can.  I added an example at he end of the README.md illustrating how one can use command substitution to do that.